### DISCUSSION TITLE
Worldwide screeners

### DISCUSSION THREAD

**Discussion by R5dan** [ranaroussi/yfinance#2189] [Yahoo's data] [answered]
Is it possible to have a screener for the whole world? (Not just 1 region?)

**Comment by ericpien** [comment, marked as answer]
I couldn't find a way to do this by checking the yahoo finance's network calls. If you can find it, that would be great.

**Reply by R5dan** [reply]
I didn't think so either as on their own builder it requires exactly one region. Just checking though.
